core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9652,6 +9666,13 @@ Sql withPostgresqlModifiedDecimalTypeSystem() {
       return dialect(postgresqlSqlDialect);
     }
 
+    Sql withClickHouseModifiedDecimalTypeSystem() {
+      final ClickHouseSqlDialect clickHouseSqlDialect =
+          new ClickHouseSqlDialect(ClickHouseSqlDialect.DEFAULT_CONTEXT
+              .withDataTypeSystem(ClickHouseSqlDialect.TYPE_SYSTEM));
```

Comment:
doesn't this DEFAULT_CONTEXT already have this TYPE_SYSTEM?

Reply:
Done，had fix here，thanks~